templates/config/overlays/namespaced/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+resources:
+- ../../default
+patches:
+- path: role.json
+  target:
+    group: rbac.authorization.k8s.io
+    version: v1
+    kind: ClusterRole
+    name: ack-sagemaker-controller
```

Comment:
Does "sagemaker" need to be hardcoded here?

Reply:
Thought I removed this must have pushed it again by accident :(

Reply:
Fair point. These might need to be made into Go templates and inject `{{ .ServiceIDClean }}` just like the `cluster-role-binding.yaml.tpl`

Reply:
Yep had to make them into go templates and put them in release.go and controller.go respectively